BHoM_Engine/Query/IsNumeric.cs

```diff
@@ -31,8 +31,9 @@ public static partial class Query
     {
         [Description("Determine whether a type is a numeric type.")]
         [Input("type", "Type that we want to check if it is numeric type or not.")]
+        [Input("enumsAreNumbers", "(Optional, defaults to true) Whether `enum` types should be considered to be numeric.")]
```

Comment:
Happy for you to keep as is, but wondering if `enumsAsNumbers` is a clearer name?

If you disagree, please feel free to ignore this comment.